### DISCUSSION TITLE
请问 jdbc catalog 执行sql，如何传递 clickhouse的关键字 FINAL 呢

### DISCUSSION THREAD

**Discussion by aruizhong** [apache/doris#23130] [A - General / Q&A] [answered]
请问 jdbc catalog 执行sql，希望clickhouse进行merge操作，select xx  final from xx，如何通过jdbc catalog传递 final关键字呢

**Comment by zy-kkk** [comment, marked as answer]
use session variable:
set jdbc_clickhouse_query_final =  true
